modules/applications/applications-nav.adoc

```diff
@@ -10,7 +10,7 @@
   *** xref:how-tos-builder.adoc[How-tos]
   *** xref:resources.adoc[Resources]
   *** xref:faq.adoc[FAQ]
- ** xref:custom-applications-index.adoc[Living applications with Bonita UID (Legacy)]
+ ** xref:uid-applications-index.adoc[Living applications with Bonita UID]
```

Comment:
Why should it be removed?

custom-applications-index.adoc and uid-applications-index.adoc are 2 distinct pages and will have different content

Reply:
custom-applications-index.adoc is the one about Legacy UID, so I don't understand what the difference would be with uid-applications-index
custom-applications-index.adoc is no longer accessible from the navigation.

I do understand the need for renaming as "custom-" is ambiguous. But if that's not a page renaming, then I don't understand the change.

Aliases are here to ensure the correct redirection in case of page renaming.

Reply:
As it is now, "Living applications" and "Living applications with Bonita UID (legacy)" are both on custom-applications-index.adoc with the same content, and I want to avoid that and split them into 2 different pages (since they will have different content). Hence this change